src/core/export/csv.js

```diff
@@ -15,8 +15,10 @@ export class Csv {
 		const values = [];
 		let head = [];
 		for (let column of columns) {
-			values.push(valueFactory(column));
-			head.push(escape(column.title));
+			if (column.key !== 'rowNumber') {
```

Comment:
Actually, I found out that for column type: "row-number" class is equal to "data"